### PR TITLE
Adds a fireball from the heavens as an admin smite

### PR DESCRIPTION
**Pull request by ghost**
ALL credit goes to coaix, I was just disheartened by the fact these never got in

Right now they can dodge it, not sure whether to put in a stun at the
same time so they can't escape.

cl: coiax and not armhulen
add: Angering the gods can cause fire to rain on the heretic.
/cl

### CODE REVIEW

**Comment by DoctoreMad** [comment]
When are you going to FINALLY get actual, biblical punishments that can't be mistaken as generic?
Pillar of Salt and Locusts are a must.

**Comment by Armhulen** [comment]
oml traaaavis

File: code/modules/admin/verbs/randomverbs.dm [hunk 1]
```diff
@@ -1193,7 +1193,9 @@ GLOBAL_LIST_EMPTY(custom_outfits) //Admin created outfits
 			target.gib(FALSE)
 		if(ADMIN_PUNISHMENT_BSA)
 			bluespace_artillery(target)
-
+		if(ADMIN_PUNISHMENT_FIREBALL)
+			new /obj/effect/overlay/temp/target(get_turf(target))
```
Comment (Cyberboss) [inline code/modules/admin/verbs/randomverbs.dm]: this isn't a type

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/modules/admin/verbs/randomverbs.dm (hunk 1 above)
Comment (RemieRichards) [reply code/modules/admin/verbs/randomverbs.dm]: /obj/effect/temp_visual/target

**Comment by AnonymousNow** [comment]
We have admin smites with animations and things? Neat.

We need thunderbolts, teleporting bombs, temporal deletion, random gibbing, dissolving into mist, acidic melting and so on. Hell, if a thunderbolt could leave a literal Zeus' Thunderbolt item imbued with the power of the smat soul, that'd be radical.

File: code/modules/admin/verbs/randomverbs.dm (hunk 1 above)
Comment (Armhulen) [reply code/modules/admin/verbs/randomverbs.dm]: Old code!

**Comment by BeeSting12** [comment]
@AnonymousNow Gibbing is already a thing, and I can already lightning bolt people.

File: code/modules/admin/verbs/randomverbs.dm (hunk 1 above)
Comment (kevinz000) [reply code/modules/admin/verbs/randomverbs.dm]: @Armhulen this is used by drakes you're going to have to change it.

Comment (Armhulen) [reply code/modules/admin/verbs/randomverbs.dm]: explain

**Comment by Armhulen** [comment]
okay everything works except for the fireball :^)

File: code/modules/admin/verbs/randomverbs.dm (hunk 1 above)
Comment (Armhulen) [reply code/modules/admin/verbs/randomverbs.dm]: oooh it don't work

**Comment by MoreRobustThanYou** [comment]
@Armhulen IIRC, ash drakes can shoot fireballs from above, steal some ashdrake code

File: code/modules/admin/verbs/randomverbs.dm [hunk 2]
```diff
@@ -1193,6 +1193,8 @@ GLOBAL_LIST_EMPTY(custom_outfits) //Admin created outfits
 			target.gib(FALSE)
 		if(ADMIN_PUNISHMENT_BSA)
 			bluespace_artillery(target)
+		if(ADMIN_PUNISHMENT_FIREBALL)
+			new /obj/effect/temp_visual/target
```
Comment (Cyberboss) [inline code/modules/admin/verbs/randomverbs.dm]: you made this in nullspace

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/modules/admin/verbs/randomverbs.dm (hunk 2 above)
Comment (Armhulen) [reply code/modules/admin/verbs/randomverbs.dm]: i just did what they said!

Comment (MoreRobustThanYou) [reply code/modules/admin/verbs/randomverbs.dm]: add ```(target.loc)```

Comment (Cyberboss) [reply code/modules/admin/verbs/randomverbs.dm]: get_turf(target)

**Comment by optimumtact** [comment]
so it doesn't work?

**Comment by Armhulen** [comment]
it works fo sho now

**Review by Cyberboss — APPROVED** [review]

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is currently below zero (-3). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

**Pull request merged**